src/app/components/demo/demo.component.html

```diff
@@ -0,0 +1,72 @@
+<div>
+  <h4>LocalStorageService Demo</h4>
+  <div class="row" *ngIf="!noProviderForLocalStorage; else noLocalStorageService">
+    <div class="col-md-6">
+      <form [formGroup]="localStorageForm">
+        <div class="form-group">
+          <label for="storageKey">Enter LocalStorage Key</label>
+          <input class="form-control" id="storageKey" type="text" formControlName="key">
+        </div>
+        <div class="form-group">
+          <label for="storageValue">Enter LocalStorage Value</label>
+          <input class="form-control" id="storageValue" type="text" formControlName="value">
+        </div>
```

Comment:
<div class="form-group">
           <label for="storageValue">Enter LocalStorage Value</label>
           <input class="form-control" id="storageValue" type="text" formControlName="value">
         </div>

can be extracted into form-input component and reused, thus economy the space

Reply:
Done